### ISSUE TITLE
Provide native executables of the CLI

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#664]
Use JReleaser's own `nativeImage` assembler.

**Comment by aalmiray** [comment]
Blocked by:

https://github.com/oracle/graal/issues/4189
https://github.com/oracle/graal/issues/2835

**Comment by aalmiray** [comment]
Binaries for MacOs and Linux are available. Closing for now. Open another ticket for Windows.
